.golangci.yml

```diff
@@ -8,6 +8,11 @@ linters:
     - ineffassign
     - misspell
     - unconvert
+    - structcheck
+    - asciicheck
+    - unused
+    - gosimple
+    - deadcode
```

Comment:
nit: can we keep this list in alphabetical order?

Reply:
Fixed!